math/Postfix_Calculator.cpp

```diff
@@ -0,0 +1,170 @@
+
+#include<iostream>
+using namespace std;
+#include<stack>
+#include<cstring>
+
+//Declaration of the functions
+void infix_to_postfix(char *in, char *post);
+bool is_operand(char ch);
+int priority(char opr);
+int postfix_evaluation(char *post);
+int calculate(int op1, int op2, char opr);
+
+//cur = current
+
+void infix_to_postfix(char *in, char *post)
+{
+	int j=0;
+	stack<int> s;
+
+	for( int i = 0 ; in[i] != '\0' ; i++ )
+	{
+		//if the cur ele is an operand
+		if( is_operand(in[i]) )
+		{
+			//append it into the postfix string
+			post[j++] = in[i];
+		}
+		else
+		if( in[i] == '(')//if the current is an opening brace
+		{
+			//push it into the stack
+			s.push(in[i]);
```

Comment:
```suggestion
			s.push_back(in[i]);
```
Same in other parts of the code.